arch/x86_64/kernel/fb/double-buffering.cpp

```diff
@@ -0,0 +1,51 @@
+#include "x86_64/fb/double-buffering.hpp"
+
+#include <stl/cstdlib/cstring.h>
+#include <stl/cstdlib/stdio.h>
+
+#include "font8x16.h"
+
+namespace firefly::kernel::tty {
+
+void DoubleBuffering::init_buffers() {
+    //Todo: Alloc buffers once we have the mm bug fixed
+    //alloc(this->front)
+    //alloc(this->back);
+
+    front.is_front = true;
+    back.is_front = false;
+
+#if DEBUG
```

Comment:
ifndef NDEBUG

Reply:
Ik ik, ignore everything related to double-buffering
I'll probably delete it for the time being